netcore/excludes-System.Net.Http.Functional.Tests.rsp

```diff
@@ -0,0 +1,8 @@
+# TODO: Crashes runtime inside Interop.Http.MultiPerform (marshalling issue?)
+-nomethod *PlatformHandler*
```

Comment:
Crash report: https://gist.github.com/filipnavara/f8a438c7e3d05b5049bc2056c35dcd47

Reply:
This may not actually be Mono's fault. It crashes inside CURL:
        0x0 - Unknown
        0x7fff62371673 - /usr/lib/libcurl.4.dylib : Curl_http_readwrite_headers
        0x7fff62388f2b - /usr/lib/libcurl.4.dylib : Curl_readwrite
        0x7fff62392a6b - /usr/lib/libcurl.4.dylib : multi_runsingle
        0x7fff6239199a - /usr/lib/libcurl.4.dylib : curl_multi_perform

Reply:
My system has "libcurl/7.54.0" installed. This looks suspiciously similar to https://github.com/curl/curl/issues/2674 which was still crash on libcurl 7.60.

Reply:
crashes for me too

Reply:
CoreFX CI seems to run the tests on macOS 10.12 and 10.13. You tried it on Mojave (10.14) too, right?